peer_test.go

```diff
@@ -392,7 +402,7 @@ func (pt *peerSelectionTest) runStressSimple(b *testing.B) {
 }
 
 func (pt *peerSelectionTest) runStress() {
-	numClock := pt.numConcurrent + pt.numAffinity
+	numClock := pt.numConcurrent + pt.numAffinity - pt.numAffinityWithNoCall
```

Comment:
we keep repeating this `pt.numAffinity - pt.numAffinityWithNoCall`. I think it might be cleaner (and less error prone) if we treat them as separate lists of affinity nodes. So you have numAffinity = 3, numAffinityWithoutCall = 4, and so there's 7 total affinity nodes. That way you can have more affinities with no call than num affinity too.